core/src/main/java/com/bumble/appyx/core/integrationpoint/requestcode/RequestCodeRegistry.kt

```diff
@@ -21,7 +20,8 @@ class RequestCodeRegistry constructor(
     private val nbLowerBitsForIds: Int = 4
 ) {
     internal val requestCodes: HashMap<Int, String> =
-        (initialState?.getSerializable(KEY_REQUEST_CODE_REGISTRY) as? HashMap<Int, String>) ?: hashMapOf()
+        (initialState?.getSerializable(KEY_REQUEST_CODE_REGISTRY) as? HashMap<Int, String>)
```

Comment:
Did you need to change this file?

Reply:
I fixed the wildcard import but the others were formatted as they exceeded the max line lenght.

Reply:
fair enough =]